zstd_vendor/CMakeLists.txt

```
@@ -59,7 +61,7 @@ macro(build_zstd)
     USE_SOURCE_PERMISSIONS)
 endmacro()
 
-if (NOT zstd_FOUND OR "${zstd_VERSION}" VERSION_LESS 1.4.4)
+if (NOT zstd_FOUND)
```

I don't think there is any reason to remove this check.  Let's just leave it:

```suggestion
if (NOT zstd_FOUND OR "${zstd_VERSION}" VERSION_LESS 1.4.4)
```

---

Yes, we do depend on specific APIs such that the version needs to be high enough